src/components/transactions/TxDetails/index.tsx

```diff
@@ -126,7 +126,9 @@ const TxDetails = ({
 
   const [txDetailsData, error, loading] = useAsync<TransactionDetails>(
     async () => {
-      trackEvent(TX_LIST_EVENTS.FETCH_DETAILS)
+      if (!txDetails) {
```

Comment:
That's a negation, isn't it? Why do we even track this event? We should track clicks on the accordeon, not the fetch.

Reply:
I also considered tracking this event on the accordion click but with the current structure it doesn't seem possible since the fetch is in a nested component and with the accordion click we can only know if it was opened or closed but not if we already fetched the data before. Do you think we should track the first accordion open in general regardless of the fetch status?

Reply:
Yes. I think we should track the user intention, not how our app interpreted it.

Reply:
I changed it so we track the accordion open once. I also renamed the event to better fit the new purpose.